tests/conftest.py

```diff
@@ -15,6 +15,14 @@
 builtins.pp = pp
 
 
+@pytest.fixture(autouse=True)
+def reset_shared_version():
+    import vitrina.structure.factories as f
```

Comment:
```suggestion
    import vitrina.structure.factories as factories
```

or 


```suggestion
    import vitrina.structure.factories as _factories
```

To not use one letter names

Reply:
Removed the function entirely